Multisig/App/SceneDelegate.swift

```diff
@@ -54,6 +55,14 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         if let userActivity = connectionOptions.userActivities.first {
             handleUserActivity(userActivity)
         }
+
+        Intercom.setApiKey(App.configuration.services.intercomApiKey, forAppId: App.configuration.services.intercomAppId)
+        Intercom.setLauncherVisible(true)
+        
+        #if DEBUG
+        Intercom.enableLogging()
+        #endif
+        Intercom.registerUnidentifiedUser()
```

Comment:
could you please extract all this intercom-related setup into a separate class similar to `FirebaseConfig.swift` and then move the setup call from this `SceneDelegate` to `AppDelegate` instead?